contracts/WildcardSteward_v3.sol

```diff
@@ -295,7 +295,7 @@ contract WildcardSteward_v3 is Initializable {
         uint256 _auctionStartPrice,
         uint256 _auctionEndPrice,
         uint256 _auctionLength
-    ) public notNullAddress(withdrawCheckerAdmin) {
+    ) public notNullAddress(_withdrawCheckerAdmin) {
```

Comment:
This had me completely stumped. Upgrade wasn't working, and it didn't even submit a transaction (so I couldn't see the error).

FML, lucky I found it. Otherwise 100% upgrade would be delayed again!